README.md

```diff
@@ -8,7 +8,136 @@ pod 'IntrepidSwiftWisdom'
 use_frameworks!
 ```
 
-####Contributions
+### Components
+
+* [Qu](#qu)
+* [After](#after)
+* [Nib Initable](#nib-initable)
+* [Color Descriptor](#color-descriptor)
+* [Cell Configuring](#cell-configuring)
+* [User Defaults](#user-defaults)
+
+#### Qu
+
+A basic wrapper for dispatch operations in Swift.  Syntax example:
+
+```Swift
+    Qu.Background {
+        // Sleep for long operation
+        sleep(4)
+        log("1")
+    } .Also {
+        sleep(4)
+        log("2")
+    } .Also {
+        sleep(1)
+        log("3")
+    } .Also {
+        sleep(1)
+        log("4")
+    } .Also {
+        sleep(1)
+        log("5")
+    } .ThenAfter(.Previous(3)) {
+        log("6: After 5, 4, & 3")
+    } .Then {
+        sleep(1)
+        log("7: After 6")
+    } .FinallyOn(.Main) {
+        sleep(1)
+        log("Finished: After All")
+    }
+```
+
+#### After
+
+A simple way to perform or repeat future operations:
+
+```Swift
+After(2.5) {
+  print("Two and a half seconds later")
+}
+
+RepeatAtInterval(1.0, numberOfTimes: 5) {
+  print("Once a second, 5 times")
+}
+```
+
+#### Nib Initable
+
+Load views from nibs:
+
+```Swift
+let myCustomView = MyCustomView.fromNib()
+```
+
+#### Color Descriptor
+
+Setup Easy Color Schemes
+
+```Swift
+enum ColorPalette : ColorDescriptor {
+    case White = "254,216,130,255"
+    case DarkGreen = "51,58,24,255"
+    case DarkGray = "64,48,56,255"
+
+    var color: UIColor {
+        return rawValue.color
+    }
+}
+```
+
+And use:
+
+```Swift
+someView.backgroundColor = ColorPalette.White.color
+```
+
+#### Cell Configuring
+
+A simple way to cover most table view registering / dequeing
+
+##### Register
+
+```Swift
+tableView.registerCell(YourCell.self)
+tableView.registerHeader(YourHeader.self)
+```
+
+##### Dequeue
+
+```Swift
+let cell: YourCell = tableView.dequeueCell(indexPath)
+//
+let header: YourHeader = tableView.dequeueHeader(section)
+```
+
+#### User Defaults
+
+A way to read and write from defaults simply and type-safe
+
+##### Define
+
+```Swift
+enum Setting : String, EnumSettingsKeyAccessible {
+    case DisplayName
+    case LastOpenDate
+}
+```
+
+##### Read
+
+```Swift
+let displayName: String? = Setting.DisplayName.readFromDefaults
```

Comment:
Maybe I'll edit the example, but that's how I'm actually using it.  In the project this was made for, the enums are private and I have something like:

```
struct ApplicationSettings {
    var displayName: String {
        get {
            return Setting.DisplayName.readFromDefaults ?? ""
        }
        set {
            Setting.DisplayName.writeToDefaults(newValue)
        }
    }
}
```

Do you think it's worth mentioning that in readme?